engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala

```diff
@@ -1,19 +1,166 @@
 package cromwell.engine.workflow
 
-import akka.actor.{Props, ActorLogging, Actor}
+import akka.actor.SupervisorStrategy.{Escalate, Directive}
+import akka.actor._
+import cromwell.backend.BackendWorkflowExecutionActor.ExecutionSucceeded
+import cromwell.core.WorkflowId
+import cromwell.engine.WorkflowSourceFiles
+import cromwell.engine.backend.WorkflowDescriptor
+import cromwell.engine.workflow.lifecycle.ExecuteWorkflowActor.{RestartWorkflow, StartWorkflow}
+import cromwell.engine.workflow.lifecycle.FinalizeWorkflowBackendsActor.{BackendFinalizationsFailed, StartFinalizations}
+import cromwell.engine.workflow.lifecycle.{FinalizeWorkflowBackendsActor, ExecuteWorkflowActor, InitializeWorkflowBackendsActor}
+import InitializeWorkflowBackendsActor.{StartInitializations, BackendInitializationsFailed, BackendInitializationsSucceeded}
+import cromwell.engine.workflow.MaterializeWorkflowDescriptorActor.{MaterializeWorkflowDescriptorFailure, MaterializationWorkflowDescriptorSuccess}
+import cromwell.engine.workflow.ShadowWorkflowActor._
+import cromwell.engine.workflow.WorkflowManagerActor.WorkflowOutputs
+import cromwell.util.AggregatedException
+import wdl4s.Call
 
 object ShadowWorkflowActor {
```

Comment:
Per earlier discussion, I think it's maybe time for this to come out of the shadows.